From 7ddec55e1399f961ca2597313a47b9a4a419a5e6 Mon Sep 17 00:00:00 2001
From: Simeon Marijon <simeon.marijon@foss.st.com>
Date: Tue, 1 Oct 2024 15:02:05 +0200
Subject: [PATCH 1551/1695] product: tfm-stm32mp2: module_reset: Add missing
 format specifier in FWK_LOG message

outlined by coverity

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 27784f1d449e172dcbb285642c3beb88680637df product: tfm-stm32mp2: module_reset: Add missing format specifier in FWK_LOG message]
Signed-off-by: Simeon Marijon <simeon.marijon@foss.st.com>
Change-Id: I254c03ae3a1929561397774d36130138b9073df2
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/SCP-firmware/+/408744
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Simeon MARIJON <simeon.marijon@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Simeon MARIJON <simeon.marijon@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 .../product/tfm-stm32mp2/module/tfm_reset/src/mod_tfm_reset.c  | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/module/tfm_reset/src/mod_tfm_reset.c b/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/module/tfm_reset/src/mod_tfm_reset.c
index ee3ff6ec4..f351238e0 100644
--- a/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/module/tfm_reset/src/mod_tfm_reset.c
+++ b/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/module/tfm_reset/src/mod_tfm_reset.c
@@ -93,8 +93,7 @@ static int reset_set_state(fwk_id_t dev_id, enum mod_reset_domain_mode mode,
         break;
     case MOD_RESET_DOMAIN_AUTO_RESET:
         FWK_LOG_DEBUG(
-            MOD_NAME "SCMI reset %u: cycle(%s)",
-            fwk_id_get_element_idx(dev_id),
+            MOD_NAME "SCMI reset %u: cycle(%s%d)",
             fwk_id_get_element_idx(dev_id),
             ctx->rstctrl->dev->name, ctx->rstctrl->id);
 ;
-- 
2.47.2

